system-test/spanner.ts

```diff
@@ -65,29 +65,45 @@ describe('Spanner', () => {
   };
   const IS_EMULATOR_ENABLED =
     typeof process.env.SPANNER_EMULATOR_HOST !== 'undefined';
+  const RESOURCES_TO_CLEAN: Array<Instance | Backup | Database> = [];
+  const DATABASE = instance.database(generateName('database'));
+  RESOURCES_TO_CLEAN.push(DATABASE);
 
   before(async () => {
     if (generateInstanceForTest) {
       await deleteOldTestInstances();
       const [, operation] = await instance.create(INSTANCE_CONFIG);
       await operation.promise();
+      RESOURCES_TO_CLEAN.push(instance);
     } else {
+      // Update 'created' label timestamp on an existing instance
+      const [meta] = await instance.getMetadata();
+      const labels = Object.assign({}, meta.labels, {created: CURRENT_TIME});
+      await instance.setMetadata({labels});
```

Comment:
Why add the "created" label?

Reply:
Thanks for the explanation! When I think of a user providing a pre-existing instance, I don't think we would ever want to delete it (unless they're just providing one that we created, then we make no such guarantee).

Reply:
Removed

Reply:
Removed the logic, thanks!